plexapi/base.py

```diff
@@ -40,23 +68,29 @@ class PlexObject:
             initpath (str): Relative path requested when retrieving specified `data` (optional).
             parent (:class:`~plexapi.base.PlexObject`): The parent object that this object is built from (optional).
     """
-    TAG = None      # xml element tag
-    TYPE = None     # xml element type
-    key = None      # plex relative url
-
-    def __init__(self, server, data, initpath=None, parent=None):
+    TAG: Optional[str] = None  # xml element tag
+    TYPE: Optional[str] = None  # xml element type
+    key: Optional[str] = None  # plex relative url
+    _INCLUDES: Dict[str, Any] = {}
+
+    def __init__(
+        self,
+        server: PlexServer,
+        data: Element,
+        initpath: Optional[str] = None,
+        parent: Optional[PlexObject] = None,
+    ):
         self._server = server
         self._data = data
         self._initpath = initpath or self.key
         self._parent = weakref.ref(parent) if parent is not None else None
-        self._details_key = None
```

Comment:
Why is this removed?

Reply:
being assigned after few lines as below and not used before that assignment and this removal, hence the statement has no effect

```py
self._details_key = self._buildDetailsKey()
```